src/core/runner.ts

```diff
@@ -309,22 +316,24 @@ export default class Runner implements RunnerInfo {
 
   async #startJourney(journey: Journey, options: RunOptions) {
     journey._startTime = monotonicTimeInSeconds();
-    this.#driver = await Gatherer.setupDriver(options);
+    this.#driver = await Gatherer.setupDriver(options, journey.type);
```

Comment:
This feels a bit weird , Maybe a better way would be to check if there are defined browser journeys and then launch the browser context etc based on that.